content/docs/addons.md

```diff
@@ -4,46 +4,46 @@ title: Add-Ons
 permalink: docs/addons.html
 ---
 
-> Note:
+> Not:
 >
-> `React.addons` entry point is deprecated as of React v15.5. The add-ons have moved to separate modules, and some of them have been deprecated.
+> `React.addons` giriş noktası React v15.5'ten itibaren kullanımdan kaldırılmıştır.. Bu eklentiler ayrı modüllere taşındı ve bazıları kullanımdan kaldırıldı.
 
-The React add-ons are a collection of useful utility modules for building React apps. **These should be considered experimental** and tend to change more often than the core.
+React eklentileri, React uygulamaları oluşturmak için faydalı ve işe yarar modüller koleksiyonudur. **Bunlar deney amaçlı düşünülmeli** ve core'dan daha sık değişime eğilimi olduğu göz önüne alınmalıdır.
```

Comment:
we can say `React'in çekirdeğinden` instead of `core`. What do you think? @GurcanH

Reply:
makes sense @alioguzhan